board/safety/safety_toyota.h

```diff
@@ -203,6 +225,19 @@ static bool toyota_tx_hook(const CANPacket_t *to_send) {
       }
     }
 
+    // SecOC cars block any form of LTA actuation for now
+    if (toyota_secoc && (addr == 0x131)) {
+      bool lta_request = GET_BIT(to_send, 3U);  // STEERING_LTA_2.STEER_REQUEST
+      bool lta_request2 = GET_BIT(to_send, 0U);  // STEERING_LTA_2.STEER_REQUEST_2
+      int lta_angle_msb = GET_BYTE(to_send, 2);  // STEERING_LTA_2.STEER_ANGLE_CMD (MSB)
+      int lta_angle_lsb = GET_BYTE(to_send, 3);  // STEERING_LTA_2.STEER_ANGLE_CMD (LSB)
+
+      bool actuation = lta_request || lta_request2 || (lta_angle_msb != 0) || (lta_angle_lsb != 0);
```

Comment:
Just a note for review: I originally wrote this to build `lta_angle`, like the other LTA message. Unfortunately the [mutation test doesn't like it](684bf3aab21b4bb6e1fcc8f52ab8e4d76c3f2c9b). We never have actuation via this path, so the angle is never valid while non-zero, so the mutation test sees it can introduce faults in the bit-shift operation without consequence.